lib/cmd/fh3/services.js

```diff
@@ -63,7 +63,7 @@ function listServiceDataSources(serviceId, cb){
 
     common.doGetApiCall(fhreq.getFeedHenryUrl(), "api/v2/services/" + service.apps[0].guid + "/data_sources", "Error reading Service data sources. ", function(err, dataSources){
       if (ini.get('table') === true) {
-        services.table = common.createTableForDataSources(dataSources || []);
+        services.table = common.createTableForDataSources(dataSources || [], service);
       }
 
       cb(err, dataSources);
```

Comment:
I know that this is not part for this ticket but if we specify invalid guid fhc will return really bad error message.
`fhc ERR! Error - not 2xx status code. (400)
....`

Maybe we can pass here some nicer string instead by using `err.message` or `err.result.message`. 
Showing entire stack is little bit confusing.